server/middleware/sanitizer.js

```diff
@@ -0,0 +1,32 @@
+import sanitize from 'sanitize-html';
+import lodashEach from 'lodash/collection/forEach';
+
+ 
+export default () => {
+  return (req, res, next) => {
+    if (req.body) {
+      req.body = sanitizeHelper(req.body);
+    }
+    return next();
+  };
+};
+
+const sanitizeHelper = (value) => {
+  if (typeof value === 'string') {
+    //value = value.replace(/<script>.*<\/script>/); // remove all script tags and anything in the middle.
+    //value = value.replace(/<.*?>/gi, ''); // for other tags, only remove the tags
+    value = value.replace(/&gt;/gi, '>');
+    value = value.replace(/&lt;/gi, '<');
```

Comment:
value.replace replaces `&gt;` tags with `>` and then html sanitizer in the next step removes all html tags